tests/utils/test_rest_utils.py

```diff
@@ -4,6 +4,7 @@
 import numpy
 import pytest
 
+import mlflow.utils.rest_utils as rest_utils
```

Comment:
```suggestion
from mlflow.utils import rest_utils
```

nit: Can we use from-import?